portal/server/controllers/dashboard/filters/query.js

```diff
@@ -0,0 +1,63 @@
+const CONSTANTS = require('../../../constants');
+const CONTENT_STRINGS = require('../../../content-strings');
+const {
+  getUserRoles,
+  isSuperUser,
+} = require('../../../helpers');
+
+const dashboardFiltersQuery = (
```

Comment:
could you add in a comment above what the query would look like? some sort of example of what to expect?

Reply:
Sure thing - just realised there's some funcitons that don't have comments/documentation at the top, will add

Reply:
Added comments to the missing functions 👍